api/apps/api/test/project-scenarios.e2e-spec.ts

```diff
@@ -325,6 +356,19 @@ async function getFixtures() {
           },
         }),
 
+    WhenAcquiringLockForScenarioAsOwner: async () =>
+      await request(app.getHttpServer())
+        .post(`/api/v1/scenarios/${scenarioId}/lock`)
+        .set('Authorization', `Bearer ${ownerToken}`),
+    WhenAcquiringLockForScenarioAsContributor: async () =>
+      await request(app.getHttpServer())
+        .post(`/api/v1/scenarios/${scenarioId}/lock`)
+        .set('Authorization', `Bearer ${contributorToken}`),
+    WhenAcquiringLockForScenarioAsViewer: async () =>
+      await request(app.getHttpServer())
+        .post(`/api/v1/scenarios/${scenarioId}/lock`)
+        .set('Authorization', `Bearer ${viewerToken}`),
+
```

Comment:
very minor, but to stick to the "rule of three" here I would suggest to extract the API call code to a `AcquireLockForScenarioAsUserWithToken(token)` and avoid the small duplication. But ok as is as well for me - it's test fixtures, not app code, and it's all code next to each other so quite easy to keep in sync in case any changes are ever needed.